python/GafferSceneUI/StandardOptionsUI.py

```diff
@@ -96,281 +97,422 @@ def __statisticsSummary( plug ) :
-			object in the scene, use a StandardAttributes node
-			with appropriate filters.
-			""",
+	"options.depthOfField" : [
 
-			"layout:section", "Motion Blur",
-			"label", "Deformation",
+		"description",
+		renderOptionsMetadata[ "depthOfField" ][1],
 
-		],
+		"layout:section", "Camera",
+	],
 
-		"options.shutter" : [
+	# Motion blur plugs
 
-			"description",
-			"""
-			The interval over which the camera shutter is open.
-			Measured in frames, and specified relative to the
-			frame being rendered.
-			""",
+	"options.cameraBlur" : [
 
-			"layout:section", "Motion Blur",
+		"description",
+		"""
+		Whether or not camera motion is taken into
+		account in the renderered image. To specify the
+		number of segments to use for camera motion, use
+		a StandardAttributes node filtered for the camera.
+		""",
 
-		],
+		"layout:section", "Motion Blur",
+		"label", "Camera",
 
-		"options.sampleMotion" : [
+	],
 
-			"description",
-			"""
-			Whether to actually render motion blur.  Disabling this
-			setting while motion blur is set up produces a render where
-			there is no blur, but there is accurate motion information.
-			Useful for rendering motion vector passes.
-			""",
+	"options.transformBlur" : [
 
-			"layout:section", "Motion Blur",
+		"description",
+		"""
+		Whether or not transform motion is taken into
+		account in the renderered image. To specify the
+		number of transform segments to use for each
+		object in the scene, use a StandardAttributes node
+		with appropriate filters.
+		""",
 
-		],
+		"layout:section", "Motion Blur",
+		"label", "Transform",
 
-		# Statistics plugs
+	],
 
-		"options.performanceMonitor" : [
+	"options.deformationBlur" : [
 
-			"description",
-			"""
-			Enables a performance monitor and uses it to output
-			statistics about scene generation performance.
-			""",
+		"description",
+		"""
+		Whether or not deformation motion is taken into
+		account in the renderered image. To specify the
+		number of deformation segments to use for each
+		object in the scene, use a StandardAttributes node
+		with appropriate filters.
+		""",
 
-			"layout:section", "Statistics",
+		"layout:section", "Motion Blur",
+		"label", "Deformation",
 
-		],
+	],
+
+	"options.shutter" : [
+
+		"description",
+		renderOptionsMetadata[ "shutter" ][1],
+
+		"layout:section", "Motion Blur",
+
+	],
+
+	"options.sampleMotion" : [
+
+		"description",
+		"""
+		Whether to actually render motion blur.  Disabling this
+		setting while motion blur is set up produces a render where
+		there is no blur, but there is accurate motion information.
+		Useful for rendering motion vector passes.
+		""",
+
+		"layout:section", "Motion Blur",
+
+	],
+
+	# Statistics plugs
+
+	"options.performanceMonitor" : [
+
+		"description",
+		"""
+		Enables a performance monitor and uses it to output
+		statistics about scene generation performance.
+		""",
+
+		"layout:section", "Statistics",
+
+	],
+
+}
+
+plugsMetadata.update( renderOptionsMetadata )
```

Comment:
Whats happening here?  The plugs on the nodes seem to be under `options.*` do we need to copy all the entires to non-prefixed plug names?

Reply:
Addressed in 1c2a4237a4d47008ec2014616029106b956c7b1c.